config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -20,12 +20,14 @@
 
 base="$(dirname $0)"
 
-# All the Kubernetes versions we're testing.
-# Must have pre-built kind images (see https://hub.docker.com/r/kindest/node/tags)
+# All the Kubernetes versions we're testing. The patch version is
+# irrelevant because the prow.sh script will pick a suitable KinD
+# image or build from source.
 k8s_versions="
-1.17.0
-1.18.0
-1.19.1
+1.17
```

Comment:
we can remove 1.17 as it's not supported anymore.

Reply:
I was wondering about that too. I kept it because I first wanted to add 1.20 (i.e. always test on three releases), but I agree that combining the removal of the old release with the addition of the new (even if not enabled immediately) cuts down the overall effort - I'll change it.

Reply:
Has this been removed yet?